generic/tests/ntttcp.py

```diff
@@ -36,11 +55,13 @@ def run(test, params, env):
     buffers = params.get("buffers").split()
     buf_num = params.get("buf_num", 200000)
     session_num = params.get("session_num")
+    timeout = int(params.get("timeout")) * int(session_num)
+    driver_name = params.get("driver_name", "netkvm")
 
     vm_sender = env.get_vm(params["main_vm"])
     vm_sender.verify_alive()
-    vm_receiver = None
-    receiver_addr = params.get("receiver_address")
+    # verify driver
+    _verify_vm_driver(vm_sender, test, driver_name, login_timeout)
```

Comment:
if has define the default timeout=360 already, no need transfer login_timeout in(due to login_timeout=360 as well).